File: programming languages/31. prova esame/Squash.java

public class Squash extends Campo {

    private final int piano;
    private final float altezza;

    public Squash(int codice, float costo, float larghezza, float lunghezza, String nome, float altezza, int piano) {
        super(codice, costo, larghezza, lunghezza, nome);
        this.altezza = altezza;
        this.piano = piano;
    }

    public float getAltezza() {
        return altezza;
    }

    public int getPiano() {
        return piano;
    }

    @Override
    public String toString() {
        return "squash\t" + super.toString() + "\t-\t-\t" + getAltezza() + "\t" + getPiano() + getCosto();
    }
}

File: Fondamenti_programmazione_II/Python/Esami/Gestione_circolo_sportivo/Squash.py
from Campo import Campo

class Squash(Campo):
    def __init__(self, codice, campo, larghezza, lunghezza, altezza, piano, costo):
        super().__init__(codice, campo, larghezza, lunghezza, costo, "squash")
        self._altezza = altezza
        self._piano = piano

    def toString(self):
        return super().__str__() + str(self._larghezza) +"\t"+ str(self._lunghezza) +"\t-\t-\t"+ str(self._altezza) +"\t"+ str(self._piano) +"\t"+ self._altezza +"\t"+ str(self._costo)